app.js

```diff
@@ -196,4 +196,22 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
     word.textContent = 'The email must be in lowercase';
   }
-});
\ No newline at end of file
+});
+
+const objectStrg = {
+  name: '',
+  mail: '',
+  msj: '',
+};
+
+form.addEventListener('click', () => {
+  objectStrg.name = document.getElementById('name').value;
+  objectStrg.mail = document.getElementById('mail').value;
+  objectStrg.msj = document.getElementById('msg').value;
+  localStorage.setItem('stored', JSON.stringify(objectStrg));
+});
```

Comment:
- [ ] Great job preserving the form information but this only happens when there is a click in the form area. Kindly change this to happen when the user changes the content of any input field in the form, the data object is updated and it is saved to the local storage. So instead of listening for a `click` event on the form, I suggest you listen for a `change` event on the input fields.